examples/bs-project/.merlin

```diff
@@ -22,10 +22,10 @@ B ./_build/*
 # the location of third-party dependencies. For us, most of our third-party deps
 # reside in `node_modules/` (made visible to Merlin through the S command
 # above); this PKG command is for discovering the opam/ocamlfind packages.
-PKG 
+PKG
 
 # FLG is the set of flags to pass to Merlin, as if it used ocamlc to compile and
 # understand our sources. You don't have to understand what these flags are for
 # now; but if you're curious, go check the rebel.ml that generated this
 # .merlin at https://github.com/reasonml/rebel
-FLG -w -30 -w -40 -open Src
+FLG -w -30 -w -40
```

Comment:
humm what happened here? The merlin logic didn't change. We just forgot to rebuild this after your recent changes right?